lookout/style/format/benchmarks/evaluate_smoke.py

```diff
@@ -0,0 +1,195 @@
+"""Module for Smoke dataset evaluation."""
+import csv
+from difflib import SequenceMatcher
+import json
+import logging
+from pathlib import Path
+import tempfile
+from typing import Any, List, Mapping
+
+from lookout.core.analyzer import ReferencePointer
+from lookout.core.api.service_analyzer_pb2 import Comment
+from lookout.core.api.service_data_pb2_grpc import DataStub
+from lookout.core.data_requests import with_changed_uasts_and_contents
+from lookout.core.lib import files_by_language, filter_files, find_new_lines
+from lookout.core.tests import server
+from lookout.style.format.analyzer import FormatAnalyzer
+from lookout.style.format.feature_extractor import FeatureExtractor
+from lookout.style.format.model import FormatModel
+from lookout.style.format.tests.test_analyzer_integration import TestAnalyzer
+
+
+def _loss(head_lines, correct_lines, predicted_lines):
+    def edit_distance(a, b):
+        err = 0
+        matcher = SequenceMatcher(a=a, b=b)
+        matcher.ratio()
+        for action, _, _, _, _ in matcher.get_opcodes():
+            if action == "equal":
+                continue
+            if action == "delete":
+                err += 1
+            elif action == "insert":
+                err += 1
+            elif action == "replace":
+                err += 2
```

Comment:
Why 2?

Reply:
because it is one delete and one incert

Reply:
Imagine that a model predicts a wrong token. It is a single mistake, not a double one.

Reply:
ok